src/slider/tp-slider.ts

```diff
@@ -222,8 +253,8 @@ export class TPSliderElement extends HTMLElement {
 		// Get subcomponents.
 		const sliderNavItems: NodeListOf<TPSliderNavItemElement> | null = this.querySelectorAll( 'tp-slider-nav-item' );
 		const sliderCounts: NodeListOf<TPSliderCountElement> | null = this.querySelectorAll( 'tp-slider-count' );
-		const leftArrow: TPSliderArrowElement | null = this.querySelector( 'tp-slider-arrow[direction="previous"]' );
-		const rightArrow: TPSliderArrowElement | null = this.querySelector( 'tp-slider-arrow[direction="next"]' );
+		const leftArrow: TPSliderArrowElement | null = this.getArrow( 'tp-slider-arrow[direction="previous"]' );
```

Comment:
@imranhsayed Can we just use `this.closest( 'tp-slider-arrow[direction="previous"]' );`?

Reply:
@junaidbhura `closest` searches up the DOM tree. The arrow is a child of the slider.
We cannot use children also because the arrow could be anywhere especially when using nested sliders, so we don't know if that arrow belongs to the parent or child slider.

Reply:
@imranhsayed How about `this.querySelector( 'tp-slider-arrow[direction="previous"]:not( tp-slider tp-slider-arrow )' );`

Reply:
Thanks @junaidbhura ,
As discussed we will come back to this later.